apps/E2E/package.json

```diff
@@ -15,6 +15,10 @@
     "url": "https://github.com/microsoft/fluentui-react-native.git",
     "directory": "apps/E2E"
   },
+  "files": [
+    "src/**/*",
+    "dist/*"
```

Comment:
We want both src and dist?

Reply:
Good question. I want to do some testing with both. If we only use the Bundle file (that includes all E2E win32 spec files), then WebDriverIO runs that in **one** session (because it's one file). Meaning the app doesn't reset after each spec file - it's sequential.

This may be ideal, but I've also run across flaky failures doing this locally. So I also want to make sure it works by just passing in file paths to each spec file within /src/